applications/imaging_ai_segmentator/README.md

```diff
@@ -1,198 +1,165 @@
+National Cancer Institute Clinical Proteomic Tumor Analysis Consortium (CPTAC). (2018). The Clinical Proteomic Tumor Analysis Consortium Cutaneous Melanoma Collection (CPTAC-CM) (Version 11) [Dataset]. The Cancer Imaging Archive. <https://doi.org/10.7937/K9/TCIA.2018.ODU24GZE>
 
+## Model
 
-## Run Instructions
+This application uses the [MONAI whole-body segmentation model](https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBody_ct_segmentation), which can segment 104 body parts from CT scans.
 
-There are a number of ways to build and run this application, as well as packaging this application as a Holoscan Application Package. The following sections describe each in detail.
+## Build and Run Instructions
 
 ### Quick Start Using Holohub Container
 
-This is the simplest and fastest way to see the application in action running as a container. The input DICOM files must first be downloaded and saved in the folder `$PWD/data/imaging_ai_segmentator/dicom`, whereas the PyTorch model is automatically downloaded when container image is built.
-
-Use the following to build and run the application:
+This is the simplest and fastest way to run the application:
 
 ```bash
-mkdir -p output
-rm -rf output/*
-./dev_container build_and_run imaging_ai_segmentator --container_args "-v $PWD/output:/var/holoscan/output -v $PWD/data/imaging_ai_segmentator/dicom:/var/holoscan/input"
+rm -rf output
+./dev_container build_and_run imaging_ai_segmentator
 ```
 
-Once the command completes, please check the output folder for the results, e.g.
-```
+**_Note:_** It takes quite a few minutes when this command is run the first time.
+
+The output will be available in the `${HOLOSCAN_OUTPUT_PATH}` directory:
+
+```console
 output
 ├── 1.2.826.0.1.3680043.10.511.3.57591117750107235783166330094310669.dcm
 └── saved_images_folder
     └── 1.3.6.1.4.1.14519.5.2.1.7085.2626
         ├── 1.3.6.1.4.1.14519.5.2.1.7085.2626.nii
         └── 1.3.6.1.4.1.14519.5.2.1.7085.2626_seg.nii
-
-2 directories, 3 files
 ```
 
-**_Note_**
-It takes quite a few minutes when this command is run the first time.
+By default, the application uses the following directories for input data, model files, and output results:
 
-### Run the Application in Dev Environment
-
-It is strongly recommended a Python virtual environment is used for running the application in dev environment.
-
-This application only has Python implementation depending on a set of Python packages from [Pypi](https://pypi.org), however, a `build_and_install` step is needed to automate organizing Python code and downloading the model.
-
-
-Set up the Holohub environment, if not already done
 ```bash
-./run setup
+HOLOSCAN_INPUT_PATH=/workspace/holohub/data/imaging_ai_segmentator/dicom
+HOLOSCAN_MODEL_PATH=/workspace/holohub/data/imaging_ai_segmentator/models
+HOLOSCAN_OUTPUT_PATH=/workspace/holohub/output
```

Comment:
Any reason why not using/modifying the origin env_settings.sh?

Reply:
These env variables need to be explicitly defined in the dockerfile so the use can set the when running the container. I think env_setting.sh is more suitable when a user wants to set up their own environment and not depending on the holohub to build and run applications. 

Previously, the default values in the dockerfile was not matching the ones defined in `env_settings.sh` but now they do. Does it make sense?

Reply:
It only make sense when a specific mode is in the context. 
- in dev mode without the dev container, these need to be set by the dev user, either directly as commands or use the shell script
- in a Holocan App package, they are not needed as the packager sets them to defaults or per spec. On CLI run, host folder maps to the values in the env vars in the container
- when dev in the dev_container, then the dev container volume mapping works.

Reply:
That's right! and it is why we kept both way of setting variables:
- for dev environment (like bare metal or other containers), the user should use `env_settings.sh` for setting them end vars as descried in `### Development Environment Setup` section
- for Holohub container the user doesn't have to set anything if they want to go with the defaults as they are being set in the container.

Reply:
These settings needs to be applied before the first reference/use (see above comment on line 65). Also, be mindful if these ones are for dev_container internal, or dev user's local env when not developing/running the app in a dev container.